controllers/topics/index.js

```diff
@@ -277,7 +309,14 @@ const _afterPutTopic = async (
       await removeTopicFromChatTab(token, name, userId);
     } else {
       await followMainFeedTopic(token, userId, name);
-      await addTopicToChatTab(token, name, userId, isAnonymous, withSystemMessage);
+      await addTopicToChatTab(
+        token,
+        name,
+        userId,
+        isAnonymous,
+        topicInvitationsId,
+        withSystemMessage
+      );
     }
   } catch (error) {
     console.log('After put topic error: ', error);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [301-319]

The function `addTopicToChatTab` has been modified to accept an additional parameter `topicInvitationsId`. This is used to handle the deletion of topic invitation channels when a user follows a topic.
```diff
-  withSystemMessage = false,
-  prevUserToken
+  topicInvitationsId,
+  prevUserToken,
+  withSystemMessage = false
...
-      await addTopicToChatTab(token, name, userId, isAnonymous, withSystemMessage);
+      await addTopicToChatTab(
+        token,
+        name,
+        userId,
+        isAnonymous,
+        topicInvitationsId,
+        withSystemMessage
+      );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->